src/constants/skip.ts

```diff
@@ -0,0 +1,33 @@
+export type RouteStatus = {
+  chainId: string | undefined;
+  txHash: string | undefined;
+  status: string | undefined;
+};
+export type SkipTransactionStatus = {
+  chainData: {
+    chainId: string | undefined;
+    chainName: string | undefined;
+    estimatedRouteDuration?: string | undefined;
```

Comment:
nit: this undefined is technically unnecessary

Reply:
yeah actually we always return a placeholder string so it wont ever even be undefined or missing